src/Adapters/Cache/RedisCacheAdapter.js

```diff
@@ -1,6 +1,8 @@
 import redis from 'redis';
 import logger from '../../logger';
 
+const DEFAULT_REDIS_TTL = 30 * 1000; // 30 seconds
```

Comment:
Don't think we use this here so could be removed.

Currently the schema cache TTL is passed in to ParseServer as schemaCacheTTL and then passed to the adapter with each put(). However InMemoryCache does have a default TTL if no value is passed in from the adapter, so you might want to implement?